src/configs/ManuscriptsEditor.ts

```diff
@@ -62,6 +61,8 @@ export interface EditorProps {
   collabProvider?: CollabProvider
   navigate: NavigateFunction
   location: Location
+  isComparingMode?: boolean
+  submissionId?: string
```

Comment:
@fwaisi , please check your PRs for unused props and arguments. This is not used and generally body editor shouldn't be aware about submissionId